extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/ResteasyReactiveDeploymentInfoBuildItem.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.resteasy.reactive.server.deployment;
+
+import org.jboss.resteasy.reactive.server.core.DeploymentInfo;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class ResteasyReactiveDeploymentInfoBuildItem extends SimpleBuildItem {
```

Comment:
Is this class actually needed? It doesn't seem to be consumed anywhere

Reply:
It's consumed in `LinksProcessor`

Reply:
Ah OK, I missed that